docs/content/guides/dagster/automating-pipelines.mdx

```diff
@@ -55,46 +55,54 @@ def configurable_job_schedule(context: ScheduleEvaluationContext):
     )
 ```
 
-### I want my data to be updated at different cadences
+### I want specify how fresh each asset needs to be and let Dagster figure out when to run them
```

Comment:
Do we need to say what fresh means?

Reply:
I think we can get away without it here

Reply:
given that we explain it in other terms in the text below